configs/job_script.default.bash

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
This file could easily be made python instead of bash but I decided to keep it similar to the slurm job script files for now.